src/IO/H5/Helpers.hpp

```diff
@@ -0,0 +1,157 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+/// \file
+/// Defines functions for h5 manipulations
+
+#pragma once
+
+#include <hdf5.h>
+#include <list>
+#include <string>
+#include <vector>
+
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Index.hpp"
+#include "IO/H5/AccessType.hpp"
+#include "IO/H5/OpenGroup.hpp"
+
+namespace h5 {
+/*!
+ * \ingroup HDF5
+ * \brief Write a "Time" attribute to the group
+ */
+void write_time(hid_t group_id, double time);
+
+/*!
+ * \ingroup HDF5
+ * \brief Write a DataVector named `name` to the group `group_id`
+ */
+template <size_t Dim>
+void write_data(hid_t group_id, const DataVector& data,
+                const Index<Dim>& extents, const std::string& name = "scalar");
+
+/*!
+ * \ingroup HDF5
+ * \brief Write the extents as an attribute named `name` to the group
+ * `group_id`.
+ */
+template <size_t Dim>
+void write_extents(hid_t group_id, const Index<Dim>& extents,
+                   const std::string& name = "Extents");
+
+/*!
+ * \ingroup HDF5
+ * \brief Write the connectivity into the group in the H5 file
+ */
+void write_connectivity(hid_t group_id, const std::vector<int>& connectivity);
```

Comment:
That the user understands how ParaView or VisIt visualizes data and what the connectivity they use is. At least those are my expectations. I can add a `\see` to the function computing connectivity if you think that would be useful.